go.mod

```
@@ -0,0 +1,26 @@
+module github.snkzt/restful-api-with-go-and-gin
```

99% of the time, this should match the path of your repository exactly.
```suggestion
module github.com/snkzt/restful-api-with-go-and-gin
```

---

😮‍💨